index.html

```diff
@@ -251,6 +399,166 @@ <h4>JSON-LD*-specific Terms</h4>
+           title="Annotating a node">
+      <div class="selectors">
+        <button class="selected input" data-selects="compacted">Compacted (Input)</button>
+        <button data-selects="expanded">Expanded (Result)</button>
+        <button data-selects="turtle">Turtle*</button>
+      </div>
+      <pre class="compacted input selected"
+           data-transform="updateExample"
+           data-options="rdfstar=true">
+      <!--
+        {
+          "@context": {
+            "@base": "http://example.org/",
+            "@vocab": "http://example.org/"
+          },
+          "@id": "bob",
+          "age": 42,
+          "@annotation": {
+            "certainty": 0.8
+          }
+        }
+      -->
+      </pre>
+      <pre class="expanded"
+           data-transform="updateExample"
+           data-options="rdfstar=true">
+      <!--
+        [{
+          "@id": "http://example.org/bob",
+          "http://example.org/age": [{"@value": 42}],
+          "@annotation": [{
+            "http://example.org/ceertainty": [{"@value": 0.8}]
+          }]
+        }]
+      -->
+      </pre>
+      <pre class="turtle"
+           data-content-type="text/turtle"
+           data-transform="updateExample"
+           data-options="rdfstar=true">
+      <!--
+      @prefix : <http://example.org/> .
+
+      :bob :age 42 .
+      << :bob :age 42 >> :certainty 0.8 .
+      -->
+      </pre>
+    </aside>
+  </section>
+
+  <section>
+    <h2>Advanced Concepts</h2>
+    <p>JSON-LD allows many more sophisticated ways of describing data and
+      both <a>embedded nodes</a> and <a>annotation objects</a> work along with these
+      other features, to the extent that they are used in describing a single relationship.</p>
+
+    <section>
+      <h3>Reverse Properties</h3>
+      <p>JSON-LD allows the description of <a data-cite="JSON-LD11#reverse-proerties"></a>
+        within <a>node objects</a>, either by defining a term using `@reverse`, or using
+        `@reverse` directly within a <a>node object</a>.</p>
+
+      <p>An <a>embedded node</a> can also use a reverse property, although
+        this is likely of limited utility:</p>
+
+      <pre class="example" title="rev1" data-ignore>TODO</pre>
+
+      <p>The relationships between nodes can be reversed even when those
+        nodes use embedded objects:</p>
+
+      <pre class="example" title="rev2" data-ignore>TODO</pre>
+
+      <p>Annotations may themselves use reverse properties, in which case
+        the annotation direction is reversed:</p>
+
+      <pre class="example" title="rev3" data-ignore>TODO</pre>
+    </section>
+
+    <p class="note"><a>Embedded nodes</a> can use a reverse property.
+      Annotating a relationship which is part of a reverse property does annotate
+      the reverse relationship, as would be expected.
+      Annotations including reverse properties effectively annotate the corresponding triple
+      as object rather than subject.
```

Comment:
```suggestion
```
This is redundant with the previous paragraph. I find it better to separate each point (reverse in annotation, annotation in reverse) in two separate paragraphs.